app/recruitment/[id]/_components/comment/section.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import {
+  CommentForm,
+  CommentList,
+} from '~/app/recruitment/[id]/_components/comment';
+import useQueryComment from '~/hooks/queries/useQueryComment';
+
+interface Props {
+  accompanyPostId: string;
+}
+
+const CommentSection = ({ accompanyPostId }: Props) => {
+  const { data: comments } = useQueryComment(accompanyPostId);
+
+  const commentsLength = comments?.length;
+
+  return (
+    <div className="w-full">
+      <h2 className="mb-4 text-xl font-semibold">댓글 {commentsLength}개</h2>
```

Comment:
혹시 comments를 구조 분해 할당으로 작성해볼 수도 있을까요? 옵셔널이 있어서 여쭈어보았어요
```suggestion
  const { length } = comments

  return (
    <div className="w-full">
      <h2 className="mb-4 text-xl font-semibold">댓글 {length}개</h2>
```